frontend/package.json

```diff
@@ -124,7 +126,18 @@
         }
       ],
       "import/newline-after-import": 1,
-      "import/no-commonjs": 0
+      "import/no-commonjs": 0,
+      "no-unused-vars": "off",
+      "unused-imports/no-unused-imports": "error",
+      "unused-imports/no-unused-vars": [
+        "warn",
+        {
+          "vars": "all",
+          "varsIgnorePattern": "^_",
```

Comment:
Ooh, does this mean I can prefix unused variables with an underscore and the linter won't yell at me for it? 😁

Reply:
Yes!! :tada: